### PR TITLE
Allow only code specific signs in code input

### PR DESCRIPTION
**Pull request by KedziaPawel**
## What does this PR do?

* allow only code specific signs in code input
* fix missing attempt try in translation 

#### *Any scout cleaning/refactoring?*
* fix code input behaviour

- [ ] Checked on iOS
- [X] Checked on Android

### CODE REVIEW

File: src/components/CodeInput.tsx [hunk 1]
```diff
@@ -12,19 +12,19 @@ interface Props {
   isError?: boolean;
 }
 
-export const CodeInput = ({ value, onTextChange, testID, isError = false }: Props) => {
-  const [_, setValue] = useState('');
+export const CodeInput: FC<Props> = ({ value, onTextChange, testID, isError = false }) => {
+  const setCodeValue = (code: string) => {
```
Comment (PanRadaCloudbest) [inline src/components/CodeInput.tsx:16]: will be good to  use "useCallback"

Comment (KedziaPawel) [reply src/components/CodeInput.tsx:16]: what is the reason that this function should use it?

Comment (PanRadaCloudbest) [reply src/components/CodeInput.tsx:16]: nvm, approved

**Review by PanRadaCloudbest — APPROVED** [review]

File: src/components/CodeInput.tsx (hunk 1 above)
Comment (KedziaPawel) [reply src/components/CodeInput.tsx:16]: thx:) https://kentcdodds.com/blog/usememo-and-usecallback based on this article the useCallback would more performance even worse in this case

Comment (PanRadaCloudbest) [reply src/components/CodeInput.tsx:16]: want to share this same link paste hire, cos see diff side this scenario, but said nvm, for not blocked this simple changes

**Pull request merged**